lib/components/__tests__/clock.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+import { Clock } from '../clock/clock';
+
+describe('clock', () => {
+    it('should render', () => {
```

Comment:
I'm second guessing what we've been testing with a `should render` test all this time. A component that returns null (erroneously or not) will pass. It we're testing that the class can be instantiated properly within the React world, then perhaps a `should mount` test is more relevant.

Although for Particle, this test would potentially be dropped because of Jest snapshots, it's worth pointing out that this test will always pass if the component can mount correctly, irrespective of its rendering.